### ISSUE TITLE
Jolt spec to convert String represents DateTime to different format

### ISSUE THREAD

**Issue by sekaisang** [bazaarvoice/jolt#490]
Hi,

I am wondering if there is any JOLT spec which can achieve this:

input:
```
{
   Update_time: 2011-09-26 21:41:51.415
}
```

output:
```
{
 Update_time: 2011-09-26T21:41:51.415Z
}

```
input2:
```
{
   Update_time: 2011-09-26 21:41:51.4
}
```

output:
```
{
 Update_time: 2011-09-26T21:41:51.400Z
}

```
input3:
```
{
   Update_time: 2011-09-26 21:41:51.45
}
```

output:
```
{
 Update_time: 2011-09-26T21:41:51.450Z
}
```

Basically, replace the space between date and time with 'T', and append 'Z' in the end. 
Also, fill the millisecond with 0 if there is less than 3 digits for millisecond. 

Thanks a lot!

**Comment by milosimpson** [comment]
Nope.

**Comment by reneli5** [comment]
You can do this:
{
  "Update_time": "2011-09-26 21:41:51.4"
}
Spec:
[
  {
    "operation": "shift",
    "spec": {
      "Update_time": {
        "*T*": {
          "$(0,1)": "test"
        }
      }
    }
  }
]
Tranforms to:
{
  "test" : "2011-09-26"
}

**Comment by G-Balamurugan** [comment]
Whether we can , change the date format

Input :
{
  "date" : "2024-03-01"
}

Output :
{
   "date" : "01/03/2024"
}
